lib/inertia_rails/configuration.rb

```diff
@@ -22,6 +22,8 @@ class Configuration
 
       # Used to detect version drift between server and client.
       version: nil,
+
+      raise_on_unoptimized_partial_reloads: false,
```

Comment:
I'd use this in production by providing a custom callback with an APM error notification call, for example. So, maybe we should accept `:error`, `:warn`, `false`, and a callable?

Reply:
It would address Brandon's reluctance to relying on warnings, and a callable would make it vary behavior based on environment (i.e. be less aggressive in prod). I like it.